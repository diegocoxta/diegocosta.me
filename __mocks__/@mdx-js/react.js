module.exports = {
  MDXProvider: jest.fn(({ children }) => children),
};
