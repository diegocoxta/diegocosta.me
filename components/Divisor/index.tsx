import styles from './styles.module.css';

export default function Divisor(): React.ReactElement {
  return <div className={styles.container} />;
}
