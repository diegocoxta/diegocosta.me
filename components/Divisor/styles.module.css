.container {
  background:
    linear-gradient(90deg, var(--main-background-color) 20px, transparent 1%) center,
    linear-gradient(var(--main-background-color) 20px, transparent 1%) center,
    var(--main-text-color);
  background-size: 22px 22px;
  height: 120px;
  margin: 40px 0;
  width: 100%;
}
