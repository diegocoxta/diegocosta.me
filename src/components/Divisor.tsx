import styled from 'styled-components';

export default styled.div`
  background:
    linear-gradient(90deg, ${({ theme }) => theme.backgroundColor} 20px, transparent 1%) center,
    linear-gradient(${({ theme }) => theme.backgroundColor} 20px, transparent 1%) center,
    ${({ theme }) => theme.textColor};
  background-size: 22px 22px;
  height: 120px;
  margin: 40px 0;
  width: 100%;
`;
