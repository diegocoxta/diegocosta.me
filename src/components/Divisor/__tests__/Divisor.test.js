import React from 'react';
import { render } from '@testing-library/react';

import Divisor from '../Divisor';

describe('<Divisor />', () => {
  it('should render properly', () => {
    const { baseElement } = render(<Divisor />);
    expect(baseElement).toMatchSnapshot();
  });
});
