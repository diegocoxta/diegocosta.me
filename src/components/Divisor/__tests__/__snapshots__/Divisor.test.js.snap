// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divisor /> renders properly 1`] = `
.c0 {
  background: linear-gradient(90deg,20px,transparent 1%) center, linear-gradient( 20px,transparent 1%) center,;
  background-size: 22px 22px;
  height: 120px;
  margin: 40px 0;
  width: 100%;
}

<body>
  <div>
    <div
      class="c0"
    />
  </div>
</body>
`;
